api/v1alpha1/iochaos_types.go

```diff
@@ -81,42 +52,27 @@ type IoChaosSpec struct {
 	// +optional
 	Value string `json:"value"`
 
-	// Duration represents the duration of the chaos action.
-	// It is required when the action is `PodFailureAction`.
-	// A duration string is a possibly signed sequence of
-	// decimal numbers, each with optional fraction and a unit suffix,
-	// such as "300ms", "-1.5h" or "2h45m".
-	// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
-	// +optional
-	Duration *string `json:"duration,omitempty"`
-
-	// Layer represents the layer of the I/O action.
-	// Supported value: fs.
-	// Default layer: fs
-	// +kubebuilder:validation:Enum=fs
-	Layer IOLayer `json:"layer"`
+	// Action defines the specific pod chaos action.
+	// Supported action: latency / fault / attrOverride
+	// +kubebuilder:validation:Enum=latency;fault;attrOverride
+	Action IoChaosType `json:"action"`
```

Comment:
`Action` is changed from `delay / errno / mixed` to `latency / fault / attrOverride`. @g1eny0ung 

Multiple IoChaos can be applied to the same pod, but they should have the same volume path.